tests/conftest.py

```diff
@@ -533,6 +536,16 @@ def activestate_oidc_service(db_session):
     )
 
 
+@pytest.fixture
+def local_file_storage(tmp_path):
+    return packaging_services.GenericLocalBlobStorage(tmp_path)
```

Comment:
Flagging: No idea if this is a good idea or not 🙂. Some other options:

1. Avoid a fixture here, inline the `GenericLocalBlobStorage` instance into the `integrity_service` fixture
2. Avoid an actual service entirely and prefer a stub/mock of an `IFileService`.

Reply:
Removed this entirely, now that we have a dev-level `NullIntegrityService`.